### ISSUE TITLE
(feature request) - Multiple status codes in savestatus

### ISSUE THREAD

**Issue by ch0wn** [tomnomnom/meg#54]
Hello,
Can you please add the option to savestatus to save multiple status codes?
Thank you

**Comment by storenth** [comment]
Yes, I think too many arguments look ugly. I suggest also add exclude option.
![meg_status](https://user-images.githubusercontent.com/18244131/89975563-48065f80-dc90-11ea-9231-55b41b9acbdb.jpeg)
